src/component/Repo/index.js

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import './style.css';
+import HoverRepo from '../HoverRepo';
+
+class Repo extends React.Component{
+    state ={
+        display: false
+    }
+    dateFormat = (date) =>{
+        const oldDate = date.slice(0, -1);
+        const dateArray = oldDate.split("T");
+
+        const newDate = `${dateArray[0]}, Time ${dateArray[1]}`
+
+        return newDate;
+    };
+    
+    hoverDisplay = () =>{
+        this.setState(prevState => ({
+            display: !prevState.display
+        }))
+    }
+
+    render(){
+        return(
+            <>
+            {this.state.display ?  (<HoverRepo 
+                display = "true" 
+                languages={(this.props.languages !== null) ? this.props.languages : 0}
+                stargazers={this.props.stargazers}
+                watchers={this.props.watchers}
+            />) : null}
+            <div className="repo" onClick={this.hoverDisplay}>
```

Comment:
I will try onHover here

Reply:
Update: onHover didn't work

Reply:
`onHover` is not a React event unfortunately! There is `onMouseEnter` though!